services/net/content/ContentManager.cs

```diff
@@ -250,8 +250,8 @@ private async Task HandleMessageAsync(ConsumeResult<string, SourceContent> resul
                     Headline = String.IsNullOrWhiteSpace(model.Title) ? "[TBD]" : model.Title,
                     Uid = model.Uid,
                     Page = "", // TODO: Provide default page from Data Source config settings.
-                    Summary = String.IsNullOrWhiteSpace(model.Summary) ? "[TBD]" : model.Summary,
-                    Body = !String.IsNullOrWhiteSpace(model.Body) ? model.Body : model.ContentType == ContentType.Snippet ? "" : model.Summary,
+                    Summary = String.IsNullOrWhiteSpace(model.Summary) ? "[TBD]" : StringExtensions.SanitizeContent(model.Summary, "<p>|</p>", "\r\n"),
```

Comment:
Does this keep the body text of the <p>?

Reply:
It only removes <p> or </p> tag and add new line without removing any content

Reply:
If we consider `<p class="XXX">`, we may need to use `<p(?:\\s[^>]*)?>|</p>` instead of `<p>|</p>`. What do you think? Also i find that there are some content with broken tags (the one Carolynn sent), it only has `<p>` without the closing tag `</p>`.

Reply:
Probably have to handle these other scenarios too.